internal/rpcserver/rpcserverhandlers_test.go

```diff
@@ -7189,6 +7196,26 @@ func TestHandleSendRawTransaction(t *testing.T) {
 		}(),
 		wantErr: true,
 		errCode: dcrjson.ErrRPCDuplicateTx,
+	}, {
+		name:    "handleSendRawTransaction: recently mined transaction",
+		handler: handleSendRawTransaction,
+		cmd: &types.SendRawTransactionCmd{
+			HexTx:         hexTx,
+			AllowHighFees: &allowHighFees,
+		},
+		mockSyncManager: func() *testSyncManager {
+			syncManager := defaultMockSyncManager()
+			syncManager.processTransactionErr = mempool.RuleError{
```

Comment:
Then, per my comment about how there will be no error in the case in question, override can go away too.

Reply:
referring to the orphan case? that would still result in some sort of real error (but I didn't check to see what error code is really returned in that case, all we need to make sure is that we are not going to pass one of the other mempool error checks, so it will query the filter for a recently confirmed tx).

Reply:
Well, I was originally referring to  it catching things in case of orphans without errors, but, as you pointed out, the RPC server passes the flag that to make it error on orphans, so there will end up being an error in that case.  I'll resolve this as it won't apply given that.